pages/docs/reference/java-to-kotlin-interop.md

```diff
@@ -397,53 +397,26 @@ public class BB8 implements Robot {
 ```
 </div>
 
-For the `@JvmDefault` annotation to take effect, the interface must be compiled with an `-Xjvm-default` argument.
-Depending on the case of adding the annotation, specify one of the argument values:
-
-* `-Xjvm-default=enabled` should be used if you add only new methods with the `@JvmDefault` annotation.
-   This includes adding the entire interface for your API.
-* `-Xjvm-default=compatibility` should be used if you are adding a `@JvmDefault` to the methods that were available in the API before.
-   This mode helps avoid compatibility breaks: all the interface implementations written for the previous versions will be fully compatible with the new version.
-   However, the compatibility mode may add some overhead to the resulting bytecode size and affect the performance.
-
-For more details about compatibility issues, see the `@JvmDefault` [reference page](/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default/index.html).
+### Compatibility mode for default methods
 
-### Using in delegates
+If there is client Java code that uses your Kotlin interfaces compiled with Kotlin 1.3 or older, then recompiling it with 
```

Comment:
The version 1.3 is not important here, what's important is that these interfaces were compiled in the old DefaultImpls-based compilation mode, which is for example still the default mode in 1.4. Maybe we could word this somehow to avoid mentioning 1.3?

Reply:
Is it correct to say _ compiled without -Xjvm-default option_? Will this be equal to _DefaultImpls-based compilation mode_?

Reply:
Technically not, because there are old modes `-Xjvm-default=enable`, `-Xjvm-default=compatibility` which are still "DefaultImpls-based" for all methods which are not annotated with the old annotation `@JvmDefault`.

But maybe something like "compiled without the _new_ -Xjvm-default options" would work? Or "compiled with the old mode (with method bodies in DefaultImpls)"